pydocstringformatter/_formatting/_utils.py

```diff
@@ -6,4 +6,6 @@ def is_rst_title(summary: str) -> bool:
     """Check if the second line of a summary is one recurring character."""
     # If second line is one recurring character we're dealing with a rst title
     last_line = summary.splitlines()[-1].lstrip()
+    if len(last_line) < 1:
```

Comment:
```suggestion
    if not last_line:
```

This should also work and save the call to `len`!